Janssen Vaccine Addition (#908)
greenelab/covid19-review: content/22.vaccines.md

GMTs for the low and high doses were slightly lower for participants ≥ 65 years old (196 and 127 respectively), potentially indicating slightly lower immunogenicity. 
Seroconversion of the S antibodies was detected in 99% of individuals between 18-55 years old for the low and high doses (GMTs 528 and 695 respectively), with similar findings reported for the ≥ 65 years old.
Indeed, both dose concentrations also induced robust Th1 cytokine-producing S-specific CD4^+^ T cells and CD8^+^ T cell responses in both age groups.  
The findings of the phase I/IIa study supported further investigation of a single immunization using the low dose vaccine. 
Therefore, 25 patients were enrolled for a second randomized double-blind, placebo-controlled phase 1 clinical trial currently being conducted in Boston, Massachusetts for 2 years [@doi:10.1001/jama.2021.3645].
Participants received either a single dose followed by a placebo, or a double dose of either a low dose (5 x 10^10^ viral particles/mL) or a high dose (1 x 10^11^ viral particles/mL) vaccine administered intramuscularly on day 1 or day 57.

> Placebo only recipients received a placebo dose on day 1 and 57. 

Reviewer: ```suggestion
Placebo-only recipients received a placebo dose on day 1 and 57. 
```

#### Janssen’s Ad26.COV2-S

> The Johnson & Johnson (J&J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&J, was conducted in collaboration with National Institute of Allergy and Infectious Diseases (NIAID), National Institutes of Health (NIH), Department of Health and Human Services (HHS), and was funded by J&J and “Operation Warp Speed”, which was run by the HHS, the Department of Defense (DOD) [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use].  

Author: ```suggestion
The Johnson & Johnson (J&J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&J, was conducted in collaboration with and funded by “Operation Warp Speed” [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use].  
```

> The Johnson & Johnson (J&J) vaccine developed by Janssen Pharmaceuticals, Inc., a subsidiary of J&J, was conducted in collaboration with National Institute of Allergy and Infectious Diseases (NIAID), National Institutes of Health (NIH), Department of Health and Human Services (HHS), and was funded by J&J and “Operation Warp Speed”, which was run by the HHS, the Department of Defense (DOD) [@url:https://www.gao.gov/products/gao-21-319; @url:https://www.jnj.com/johnson-johnson-announces-a-lead-vaccine-candidate-for-covid-19-landmark-new-partnership-with-u-s-department-of-health-human-services-and-commitment-to-supply-one-billion-vaccines-worldwide-for-emergency-pandemic-use].  

Author: Fair point, amended

This allows the vaccine to be distributed easily without the requirement for very low temperature storage, unlike many of the other COVID-19 vaccine candidates.
J&J screened numerous potential vaccine candidates _in vitro_ and in animal models using varying different designs of the S protein, heterologous signal peptides, and prefusion-stabilizing substitutions [@doi:10.1038/s41541-020-00243-x]. 
A select few candidates were further investigated as a single dose regimen in Syrian golden hamsters, a single dose regimen in rhesus macaques, and a single- and two-dose regimen in both adult and aged rhesus macaques [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].
From these studies, the JNJ-78436735 candidate was selected for its favorable immunogenicity profile and ease of manufacturability [@doi:10.1038/s41541-020-00243-x; @doi:10.1038/s41586-020-2607-z; @doi:10.1038/s41591-020-1070-6; @doi:10.1101/2020.11.17.368258].

> A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10^9^ viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that higher doses of 1.125 x 10^10^ were required to close achieve close to complete protection in nasal swabs  [@doi:10.1101/2021.01.27.428380]. 

Author: ```suggestion
A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10^9^ viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that doses higher than 1.125 x 10^10^ were required to close achieve close to complete protection in nasal swabs  [@doi:10.1101/2021.01.27.428380]. 
```

> A SARS-CoV-2 challenge study in rhesus macaques showed that vaccine doses as low as 2 x 10^9^ viral particles/mL was sufficient to induce strong protection in bronchoalveolar lavage but that higher doses of 1.125 x 10^10^ were required to close achieve close to complete protection in nasal swabs  [@doi:10.1101/2021.01.27.428380]. 

Author: Corrected, thanks for spotting @rando2

Indeed, six months post-immunization, levels of S-binding and neutralizing antibodies in rhesus macaques indicated that the JNJ-78436735 vaccine conferred durable protection against SARS-CoV-2 [@doi:10.1101/2021.01.30.428921].

Following selection of the JNJ-78436735 vaccine, J&J began phase I/IIa trials. 
The interim phase I/IIa data was placed on the _medRxiv_  preprint server on September 25th, 2020 [@doi:10.1101/2020.09.23.20199604], which was later published in the _New England Journal of Medicine_ on January 13th, 2021 [@doi:10.1056/NEJMoa2034201].
The phase I/IIa multi-center, randomized, placebo-controlled trial enrolled 402 healthy participants between 18-55 years old and a further 403 healthy older participants ≥ 65 years old [@doi:10.1056/NEJMoa2034201].

> A dose of either 5 x 10^10^ (low dose) or 1 X 10^11^ (high dose) viral particles per mL was administered intramuscularly as part of either a single dose or two-dose regimen 56 days apart. 

Author: I think it is correct to say 2 x 2 design, but maybe the placebo needs to be incorporated into this section. Each participant had two visits... randomized to either a low-dose or high-dose. On the second visit.. the participants were given either the same vaccine they got previously, or a placebo vaccine. Is that clear from what is written,  or do we need some editing haha

> A dose of either 5 x 10^10^ (low dose) or 1 X 10^11^ (high dose) viral particles per mL was administered intramuscularly as part of either a single dose or two-dose regimen 56 days apart. 

Author: I doubled checked, there are 5 possible outcomes in this trial: low dose followed by low dose, low dose followed by placebo, high dose followed by high dose, high dose followed by placebo, and placebo followed by placebo

> A dose of either 5 x 10^10^ (low dose) or 1 X 10^11^ (high dose) viral particles per mL was administered intramuscularly as part of either a single dose or two-dose regimen 56 days apart. 

Author: What is the best way to convey that message... or do we even need that level of detail...

> A dose of either 5 x 10^10^ (low dose) or 1 X 10^11^ (high dose) viral particles per mL was administered intramuscularly as part of either a single dose or two-dose regimen 56 days apart. 

Reviewer: Ah! Maybe something like this? (I may also just be bad at reading, but I always get lost easily when there are a bunch of different conditions)

```suggestion
Patients were administered either a placebo, a low dose (5 x 10^10^ viral particles per mL), or a high dose (1 X 10^11^ viral particles per mL) intramuscularly as part of either a single- or two-dose regimen.
All patients received injections 56 days apart, but participants in the single-dose condition received the placebo at the second appointment.
```

Those who received only one dose of either vaccine received a placebo dose at their second vaccination visit. 
A comparison of the single versus the double dose regimen has yet to be published. 
The primary endpoints of both the trial were safety and reactogenicity of each dose. 
Fatigue, headache, myalgia, and pain at the injection site were the most frequent solicited adverse events reported by participants. 
Although less common, particularly for those in the elderly cohort and those on the low dose regimen, the most frequent systemic adverse effect was fever. 
Overall, immunization was well tolerated, particularly at the lower dose concentration.

In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization. 

> Geometric mean titres (GMT) between 224-354 were detected regardless of age and the GMT reached 100% by day 57 (GMT 288-488) and remained stable until day 71.

Author: Ya I'm not sure we need it either. GMT is a way of expressing antibody titers. It is just a normalized way of expressing the antibody titers as there is huge variability between individuals, labs, and methods used. The higher the better. Sorry it was the way it was phrased was poor. It is a confusing paper to unravel. Let me know if this makes more sense. 
```suggestion
Neutralizing geometric mean ratio of antibody titers (GMT) between 224-354 were detected regardless of age. 
By day 57,  100% of the 18-55 year old participants had neutralizing GMT (288-488), which remained stable until day 71.
```

> Geometric mean titres (GMT) between 224-354 were detected regardless of age and the GMT reached 100% by day 57 (GMT 288-488) and remained stable until day 71.

Reviewer: This seems good to me! I was thinking I should go through and try to add a little background to the intro about how efficacy is assessed. During editing, we might want to double-check how we report it across all the different sections to try to make consistent comparisons. So I may ping you on this again down the road as we flesh that part of it out!

> In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization. 

Author: I put it in as it is one of the main endpoints for reactogenicity... but we can use more sweeping statements generally about the endpoints to reduce content if necessary

> In terms of reactogenicity, over 90% of those who received either the low or high dose demonstrated seroconversion in a neutralization assay using wild-type SARS-CoV-2, 29 days after immunization. 

Reviewer 2: It's probably okay to leave it in at this stage.  It can be easier to have a comprehensive draft merged in and then assess whether we want to be more concise anywhere.

> In the ≥ 65 years old cohort, only 15 were assessed for seroconversion where 100% and 83% seroconversion was observed for the low and high doses. 

Author: I will check that out, it might be from the press release.

> In the ≥ 65 years old cohort, only 15 were assessed for seroconversion where 100% and 83% seroconversion was observed for the low and high doses. 

Reviewer: I am also just so burned out today that it's not clear I remember how to read 😆 It was a very long weekend but hopefully the other COVID project I've been working on is done now 🤞

> In the ≥ 65 years old cohort, only 15 were assessed for seroconversion where 100% and 83% seroconversion was observed for the low and high doses. 

Author: It was out of date... but somewhat the same... I'm not sure why that is... Maybe there is only so much of a response that the aging immune system can manage. They confirmed their finding in the supplementary material. The general reporting of this is on page 7 of the NEJM paper... 
They do report GMT for the second shot of both cohorts, but I purposely left it out as I though the paper was confusing enough without adding more... what do you think

```suggestion
In the ≥ 65 years old cohort, the incidence of seroconversion for the low- and high-dose was 96% and 88% respectively by day 29. 

```

> In the ≥ 65 years old cohort, only 15 were assessed for seroconversion where 100% and 83% seroconversion was observed for the low and high doses. 

Reviewer: That seems good to me! I think I need to read a bit more about how they measure efficacy, I understand some of the measures but others I think I just need to reflect on for a bit!

GMTs for the low and high doses were slightly lower for participants ≥ 65 years old (196 and 127 respectively), potentially indicating slightly lower immunogenicity. 
Seroconversion of the S antibodies was detected in 99% of individuals between 18-55 years old for the low and high doses (GMTs 528 and 695 respectively), with similar findings reported for the ≥ 65 years old.
Indeed, both dose concentrations also induced robust Th1 cytokine-producing S-specific CD4^+^ T cells and CD8^+^ T cell responses in both age groups.  
The findings of the phase I/IIa study supported further investigation of a single immunization using the low dose vaccine. 
Therefore, 25 patients were enrolled for a second randomized double-blind, placebo-controlled phase 1 clinical trial currently being conducted in Boston, Massachusetts for 2 years [@doi:10.1001/jama.2021.3645].

> Participants received either a placebo, single dose, or double dose of either a low dose (5 x 10^10^ viral particles/mL) or a high dose (1 x 10^11^ viral particles/mL) vaccine administered intramuscularly on day 1 or day 57.

Author: I may have just confused things further... but... 
```suggestion
Participants received either a single dose followed by a placebo, or a double dose of either a low dose (5 x 10^10^ viral particles/mL) or a high dose (1 x 10^11^ viral particles/mL) vaccine administered intramuscularly on day 1 or day 57.
Placebo only recipients received a placebo dose on day 1 and 57. 
```

Indeed, CD4^+^ and CD8^+^ T-cell responses were also induced, which may provide additional protection, particularly if antibodies wane or poorly respond to infection [10.1038/s41586-020-03041-6]. 

On September 23rd, 2020, J&J launched its phase III trial ENSEMBLE and released the study protocol to the public [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate; @clinicaltrials: NCT04505722].
The trial intended to enroll 60,000 volunteers to assess the safety and efficacy of the single vaccine dose versus placebo with primary endpoints of 14 and 28 days post-immunisation [@url:https://www.jnj.com/johnson-johnson-initiates-pivotal-global-phase-3-clinical-trial-of-janssens-covid-19-vaccine-candidate].
The trial was conducted in Argentina, Brazil, Chile, Colombia, Mexico, Peru, South Africa, and the U.S.
The trial was paused briefly in October 2020 to investigate a “serious medical event”, but resumed shortly after [@url:https://www.jnj.com/our-company/johnson-johnson-prepares-to-resume-phase-3-ensemble-trial-of-its-janssen-covid-19-vaccine-candidate-in-the-us].

> An interim analysis was reported via press release on January 29th, 2021 [@url:https://www.nih.gov/news-events/news-releases/janssen-investigational-covid-19-vaccine-interim-analysis-phase-3-clinical-data-released; @url:https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf].

Author: Super, I will see if I can get another link if it doesn’t work

The interim data included 43,783 participants that accrued 468 symptomatic cases of COVID-19. 
It was reported that the JNJ-78436735 vaccine was 66% effective across all regions studied for the prevention of moderate to severe COVID-19 28 days post-vaccination in those aged 18 years and older. 
Notably, JNJ-78436735 was 85% effective for the prevention of laboratory-confirmed severe COVID-19 and 100% protection against COVID-19-related hospitalization and death 28 days post-vaccination across all study sites.

> Efficacy of the vaccine increased over time and no symptomatic cases were reported 49 days post-vaccination. 

Author: ```suggestion
Efficacy of the vaccine against severe COVID-19 increased over time and there were no cases of COVID-19 reported in participants after day 49 post-vaccination. 
```

> Efficacy of the vaccine increased over time and no symptomatic cases were reported 49 days post-vaccination. 

Author: I think this is clearer now.

The vaccine was well tolerated and consistent with previous vaccines produced using the AdVac^®^ platform. 
Fever occurred in 9% of vaccine recipients, with grade 3 fever occurring in only 0.2% of recipients. 
Serious adverse events were reportedly higher in the placebo group than the vaccine group, and no anaphylaxis was reported [@url:https://www.janssen.com/emea/sites/www_janssen_com_emea/files/johnson_johnson_announces_single-shot_janssen_covid-19_vaccine_candidate_met_primary_endpoints_in_interim_analysis_of_its_phase_3_ensemble_trial.pdf]. 

At the time the phase III trial was being conducted, several concerning variants, including the B.1.1.7 [@url:https://cov-lineages.org/global_report_B.1.1.7.html] and B.1.351 [@url:https://cov-lineages.org/global_report_B.1.351.html], were spreading across the globe. 
In particular, B.1.351 was first identified in South Africa, which was one of the JNJ-78436735 vaccine trial sites.
Therefore, the J&J investigators also analyzed the efficacy of the JNJ-78436735 vaccine associated with their various trial sites to determine any potential risk of reduced efficacy as a result of the novel variants. 

> It was determined that JNJ-78436735 was 72% effective in the U.S., 66% effective in Latin America, and 57% effective in South Africa, 28 days post-vaccination.

Reviewer: ```suggestion
It was determined that JNJ-78436735 was 72% effective in the U.S., 66% effective in Latin America, and 57% effective in South Africa 28 days post-vaccination.
```
